components/blocks/nav/SiteNavbar/SiteNavbar.vue

```diff
@@ -28,69 +17,28 @@
               d="M12 8v4l3 3m6-3a9 9 0 11-18 0 9 9 0 0118 0z"
             />
           </svg>
-          <h1
-            class="
-              w-full
-              py-2
-              text-lg text-center
-              md:text-left
-              font-semibold
-              leading-6
-              tracking-tight
-              hidden
-              md:inline
-            "
-          >
-            speedrun.website
-          </h1>
+          <h1>leaderboards.gg</h1>
         </NuxtLink>
       </div>
-      <div
-        :class="{ hidden: !mobileNavIsActive }"
-        class="
-          lg:flex
-          border-t border-gray-300
-          lg:border-t-0
-          flex-col
-          lg:flex-row
-          mt-2
-          lg:mt-0
-          w-full
-          lg:w-auto
-          space-y-2
-          lg:space-y-0
-        "
-      >
-        <CoreNavLinks />
-        <CoreSearchBar />
+      <!--      mobile menu when click hamburger  -->
+      <div :class="{ hidden: !mobileNavIsActive }" class="mobile-menu">
```

Comment:
💭 If we need this comment still, maybe a more descriptive class name would be better?

Reply:
I think splitting the mobile nav out of the component is also a possible route that could be good. And I think it will also be easier to read once I get all the SVGs out